contracts/SpokePool.sol

```diff
@@ -332,11 +337,17 @@ abstract contract SpokePool is
         // buffer of this contract's block time to allow for this variance.
         // Note also that quoteTimestamp cannot be less than the buffer otherwise the following arithmetic can result
         // in underflow. This isn't a problem as the deposit will revert, but the error might be unexpected for clients.
+
+        //slither-disable-next-line timestamp
         require(
             getCurrentTime() >= quoteTimestamp - depositQuoteTimeBuffer &&
                 getCurrentTime() <= quoteTimestamp + depositQuoteTimeBuffer,
             "invalid quote time"
         );
+
+        // Increment count of deposits so that deposit ID for this spoke pool is unique.
```

Comment:
Move the storage variable modification to `numberOfDeposits` above the external call `safeTransferFrom` to align with CEI pattern